templates/compositions/ec-component-content-item/content-item.html.twig

```diff
@@ -0,0 +1,123 @@
+  title: {},
+  image: '',
+  image_position: 'before',
+  infos: [],
+  tags: [],
+  links: []
+} %}
+
+{% if content_item is defined %}
+  {% set _content_item = _content_item|merge(content_item) %}
+{% endif %}
+
+{% set _css_class = 'ecl-content-item ecl-u-d-flex ecl-u-border-bottom ecl-u-border-color-grey-15 ecl-u-pv-m' %}
+{% set _extra_attributes = '' %}
+
+{# Internal logic - Process properties #}
+
+{% if extra_classes is defined and extra_classes is not empty %}
+  {% set _css_class = _css_class ~ ' ' ~ extra_classes %}
+{% endif %}
+
+{% if extra_attributes is defined and extra_attributes is not empty and extra_attributes is iterable %}
+  {% for attr in extra_attributes %}
+    {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value ~ '"' %}
+  {% endfor %}
+{% endif %}
+
+{# Print the result #}
+
+<article class="{{ _css_class }}"{{ _extra_attributes|raw }}>
+  {% if _content_item.image and _content_item.image_position == 'before' %}
+    <div role="img" aria-label="{{ _content_item.image.alt }}" class="ecl-u-ratio-3-2 ecl-u-flex-shrink-0 ecl-u-mr-xl ecl-u-d-lg-block"
+         style="background-image:url(&quot;{{ _content_item.image.src }}&quot;);background-size:contain;height:5rem;width:7.5rem"></div>
+    <div role="img" aria-label="{{ _content_item.image.alt }}"
+         class="ecl-u-ratio-3-2 ecl-u-flex-shrink-0 ecl-u-mr-xl ecl-u-d-none ecl-u-d-lg-none"
+         style="background-image:url(&quot;{{ _content_item.image.src }}&quot;);background-size:contain;height:8.75rem;width:13.125rem">
+    </div>
+  {% endif %}
+  <div class="ecl-u-flex-grow-1">
+    {% if _content_item.meta is not empty %}
+      <div class="ecl-content-item__meta ecl-u-type-s ecl-u-type-color-grey-75">{{ _content_item.meta|join(" | ") }}</div>
+    {% endif %}
+    <div class="ecl-content-item__title ecl-u-type-prolonged-m ecl-u-type-bold ecl-u-mt-xs">
+      {%- if _content_item.title.path %}
+        {% include '../ec-component-link/link.html.twig' with {
+          link: _content_item.title
+        } only %}
+      {% else %}
+        {{- _content_item.title.label }}
+      {%- endif -%}
+    </div>
+    <div class="ecl-content-item__description ecl-u-type-paragraph ecl-u-type-color-grey-100 ecl-u-mt-xs">{{ _content_item.description }}</div>
+    {% if _content_item.infos is defined and _content_item.infos is not empty and _content_item.infos is iterable %}
+      <ul class="ecl-content-item__info ecl-u-mt-m ecl-unordered-list ecl-unordered-list--no-bullet">
+        {% for info in _content_item.infos %}
+          <li class="ecl-content-item__info-item ecl-u-d-flex ecl-u-align-items-center ecl-u-type-color-grey-75 ecl-unordered-list__item">
+            {% include '../ec-component-icon/icon.html.twig' with {
+              icon: info.icon|merge({ size: 'm' })
+            } only %}
+            <span class="ecl-u-type-s ecl-u-ml-s">{{ info.label }}</span>
+          </li>
+        {% endfor %}
+      </ul>
+    {% endif %}
```

Comment:
We don't support `infos` yet right? Let's remove this part for the moment, and any other reference to that.

Reply:
I confirm that the pattern doesn't support the `infos` yet. I prefer to keep the same template (but resolving bugs) that ECL.
But I will remove this reference.